views/partials/dashboards/breaches-dash.hbs

```diff
@@ -36,9 +36,10 @@
       <h2 class="pref-headline">{{ getString "email-addresses-title" }}</h2>
       {{> dashboards/manage-email-link variableClass="hide-mobile"}}
     </div>
-    <div class="email-cards flx flx-col jst-cntr">
+    <div class="email-cards flx flx-col jst-cntr {{#if experimentFlags.treatmentBranch}}experiment{{/if}}">
       {{#each this.verifiedEmails }}
         {{> email-card }}
+        {{> experiment--share-promo-dashboard}}
```

Comment:
Due to current Express logic limitations, this prints after EVERY email a user is monitoring. Through CSS, we're hiding them all by default (`display: none`) and only showing the first one.

Reply:
@lesleyjanenorton Thoughts?